packages/cadl-ranch-api/src/request-validations.ts

```diff
@@ -132,3 +132,32 @@ const coerceDate = (targetObject: Record<string, unknown>): Record<string, unkno
   stringRep = stringRep.replace(/(\d\d\d\d-\d\d-\d\d[Tt]\d\d:\d\d:\d\d)(\.\d{3,7})?([Zz]|[+-]00:00)/g, "$1Z");
   return JSON.parse(stringRep);
 };
+
+/**
+ * Check whether the value follow the right format.
+ */
+export const validateValueFormat = (value: string, format: "uuid" | "rfc7123" | "rfc3339"): void => {
+  switch (format) {
```

Comment:
Will we add more format validation in future?

Reply:
If new format check is needed, then we could add more format.